src/lib/transition_router/initial_validator.ml

```diff
@@ -60,20 +63,19 @@ module Make (Inputs : Inputs_intf) :
         in
         let sender = Envelope.Incoming.sender transition_env in
         match%map
-          Protocol_state_validator.validate_consensus_state ~time_received
-            transition
+          Protocol_state_validator.validate_consensus_state ~logger
+            ~trust_system ~time_received ~sender transition
         with
         | Ok verified_transition ->
             ( `Transition
                 (Envelope.Incoming.wrap ~data:verified_transition ~sender)
             , `Time_received time_received )
             |> Writer.write valid_transition_writer
-        | Error e ->
+        | Error () ->
             Logger.warn logger ~module_:__MODULE__ ~location:__LOC__
               ~metadata:
                 [ ("peer", Envelope.Sender.to_yojson sender)
-                ; ("error", `String (Error.to_string_hum e))
                 ; ("transition", External_transition.to_yojson transition) ]
-              !"Got an invalid transition from $peer: $error" )
+              !"Failed to validate transition from $peer" )
```

Comment:
`Protocol_state_validator.validate_consensus_state` already punishes in this case. I agree, the code is very messy. I am working on cleaning up all the validation stuff in another PR. I didn't want to address any of the structural issues regarding validation in this PR since I'm about to redo it all.